contracts/Ownable.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.8.0;
+
+import "./external/openzeppelin/Context.sol";
```

Comment:
✂️  we don't need this

Reply:
Done: https://github.com/pooltogether/owner-manager-contracts/pull/2/commits/bc63dde7817d9fba3f8b8d8344a61bdaa1a728ba